### PR TITLE
Do not rely on loadedmetadata event for media player

### PR DESCRIPTION
**Pull request by christianmemije**
# Details

### Summary

* Meant to address https://github.com/fle-internal/kolibri-instant-schools-plugin/issues/116
* Basically, the `loadedmetadata` event is not fired on some mobile browsers until the video is played. See https://stackoverflow.com/a/21194537
* I tested this on the device, and sure enough, the `loadedmetadata` event was only fired until the video was played. 
* So instead of waiting for this event to fire, we instead just call the handleReadyPlayer method from within the videojs callback.

### Reviewer guidance

* You can test on the Vodacom Smart Kicka ve (A5).
* Also, just test in your regular browser and make sure the video player is initiated properly. 

### References

* Fixes https://github.com/fle-internal/kolibri-instant-schools-plugin/issues/116

# Contributor Checklist

- [x] PR has the correct target milestone
- [x] PR has the appropriate labels
- [x] Changes in the PR do not introduce accessibility regressions ([confimed by testing with one of the recommended tools](http://kolibri.readthedocs.io/en/develop/dev/manual_testing.html#accessibility-a11y-testing)) 
- [x] If PR is ready for review, it has been assigned or requests review from someone
- [x] ~Documentation is updated as necessary~ N/A
- [x] ~External dependency files are updated (`yarn` and `pip`)~ N/A
- [x] ~If internal dependency is updated, link to diff is included~ N/A
- [x] ~Screenshots of any front-end changes are in the PR description~ Video shows up, that's it.
- [x] ~CHANGELOG.rst is updated for high-level changes~ Is this worth an addition?
- [x] ~You've added yourself to AUTHORS.rst if you're not there~ N/A

# Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] PR has been fully tested manually

### CODE REVIEW

File: kolibri/plugins/media_player/assets/src/views/index.vue [hunk 1]
```diff
@@ -212,8 +212,7 @@
         }
 
         this.$nextTick(() => {
-          this.player = videojs(this.$refs.player, videojsConfig);
-          this.player.on('loadedmetadata', this.handleReadyPlayer);
+          this.player = videojs(this.$refs.player, videojsConfig, () => this.handleReadyPlayer());
```
Comment (indirectlylit) [inline kolibri/plugins/media_player/assets/src/views/index.vue]: any reason not to use

```javascript
this.player = videojs(this.$refs.player, videojsConfig, this.handleReadyPlayer);
```

**Comment by indirectlylit** [comment]
any idea why we were waiting for that event initially? Maybe for the `loading = false` update?

**Comment by christianmemije** [comment]
> any idea why we were waiting for that event initially? Maybe for the loading = false update?

Traced it back to this commit

https://github.com/learningequality/kolibri/commit/daa7cb8bc72764a166f1feebb06247be7dcf393e#diff-b5c693ef82f9fa7bee745c3f78b35516

I looks like @rtibbles added this to calculate the aspect ratio from the video. That info is part of the metadata. But now we just use 16:9 as the default aspect ratio to keep the video player the same aspect ratio, regardless of the video's aspect ratio.

File: kolibri/plugins/media_player/assets/src/views/index.vue (hunk 1 above)
Comment (christianmemije) [reply kolibri/plugins/media_player/assets/src/views/index.vue]: Nope :disappointed: Will make change.

Comment (christianmemije) [reply kolibri/plugins/media_player/assets/src/views/index.vue]: @indirectlylit Updated :racehorse:

**Pull request merged**